reana_server/app.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with REANA; if not, see <http://www.gnu.org/licenses>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+"""Reana-Server."""
+
+import logging
+from flask import Flask
+
+app = Flask(__name__)
+app.secret_key = "hyper secret key"
+
+
+@app.route('/ping', methods=['GET'])
+def ping():  # noqa
+    r"""Endpoint to ping the server. Responds with a pong.
```

Comment:
Shall we plug `apispec` and `sphinxcontrib-openapi` already as here https://github.com/reanahub/reana-job-controller/pull/40? We will need it if we want to show API docs...
If we do so, take into account pinning `Sphinx` see https://github.com/reanahub/reana-job-controller/pull/43